src/main/java/mate/academy/service/UserService.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.service;
+
+import java.util.Optional;
+import mate.academy.model.User;
+
+public interface UserService {
+    User add(User user);
+
+    Optional<User> findByEmail(String email);
```

Comment:
Service should handle null case. Service should not return Optional

Reply:
This service uses Optional according to the terms of the task in the Readme file